### PR TITLE
Dnd fixes

### PR DESCRIPTION
**Pull request by sbrubes**
That's it.

### CODE REVIEW

**Comment by sbrubes** [comment]
@virtuald I did not test the `text/uri-list` since I'm on mac, and will be handle by #480

File: xlgui/widgets/playlist.py [hunk 1]
```diff
@@ -1295,17 +1299,38 @@ def on_drag_data_received(self, widget, context, x, y, selection, info, etime):
...
                         self.playlist.extend(tracks)
-
-                    return
+                        insert_position = len(self.playlist) - len(tracks)
                 except Exception:
                     pass  # Any exception on hack is ignored
+                else:
+                    # Select inserted items
+                    if tracks:
+                        self.get_selection().select_range(
```
Comment (virtuald) [inline xlgui/widgets/playlist.py]: Seems like you should unselect prior selected tracks?

**Comment by sbrubes** [comment]
@virtuald I could not reproduce that.

Are you sure shuffle isn't on?

File: xlgui/widgets/playlist.py (hunk 1 above)
Comment (sbrubes) [reply xlgui/widgets/playlist.py]: You are probably right, but it seems to be working (maybe it unselects or moves cursor when model changes?).

Comment (virtuald) [reply xlgui/widgets/playlist.py]: No, the old tracks are selected along with the newly dragged tracks, which isn't very intuitive.

**Comment by virtuald** [comment]
Nevermind, I had things in the queue... it was playing from the queue instead. Weird.

File: xlgui/widgets/playlist.py (hunk 1 above)
Comment (virtuald) [reply xlgui/widgets/playlist.py]: I think if you unselect tracks before you select new tracks, this PR is ready to go.

Comment (sbrubes) [reply xlgui/widgets/playlist.py]: > No, the old tracks are selected along with the newly dragged tracks, which isn't very intuitive.

Are you saying that you actually have this behavior?

**Comment by sbrubes** [comment]
Sorry, I’ve forgot to test/check Queue.

**Pull request merged**